### ISSUE TITLE
Bug - MS Teams - Unable to send messages to the chatbot on MS Teams Channel

### ISSUE THREAD

**Issue by ShefaliKukreja** [MicrosoftDocs/msteams-docs#10669]
### Steps to reproduce

We are unable to send messages to the chatbot that is developed using Azure Bot service via MS Teams.
This chatbot is not sideloaded or published to the org in MS Teams but is being used via the MS Teams channel link from azure portal.

<img width="642" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/135143999/809f8ce6-45f4-45c9-a62f-f948f7b84d26">

### Expected behavior

Should be able to send messages to the chatbot via MS Teams channel without publishing the bot as MS Teams app.
This use to work fine previously. We are trying to communicate with the chatbot in test environment and don't really want to publish the chatbot as MS Teams apps for test/dev environment.

### Actual behavior

Unable to communicate with the bot as the option to send messages appears grayed out.

**Comment by Prasad-MSFT** [comment]
Hi @ShefaliKukreja - Is it happening in New Teams or Teams Classic?

**Comment by ShefaliKukreja** [comment]
> Hi @ShefaliKukreja - Is it happening in New Teams or Teams Classic?

New Teams

**Comment by Prasad-MSFT** [comment]
@ShefaliKukreja - We tried to repro this issue using chatbot that is developed using Azure Bot service via MS Teams. As mentioned, the chatbot is not sideloaded or published to the org in MS Teams but is being used via the MS Teams channel link from azure portal. However, we did not encounter the issue. We are able to communicate with the bot successfully.
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/62237649/afe8b8dc-97cb-4c59-ba40-e038adc4c118)

```
Microsoft Teams version 24093.1900.2795.5438.
The client version is 49/24032122305.
```
